scripts/disableDepositRoutes.ts

```diff
@@ -44,11 +51,11 @@ export async function run(logger: winston.Logger): Promise<void> {
     originToken: string;
     depositsEnabled: boolean;
   }[] = [];
-  for (const chainId of config.chainIdListIndices) {
+  for (const chainId of enabledChains) {
     const depositRoutesForChain = spokePoolClients[chainId].getDepositRoutes();
     for (const originToken of Object.keys(depositRoutesForChain)) {
-      // If chainId is the one we want to disable, disable every route to every other chain from it.
-      if (chainId === chainToDisable) {
+      // If origin chainId is the only one we want to disable, disable every route to every other chain from it.
```

Comment:
Wouldn't this case be hit in the disable all chains case too?
```suggestion
      // If we want to disable this chainId, disable every route to every other chain from it.
```